### ISSUE TITLE
 Requirements for gardena_smart_system not found: ['py-smart-gardena==1.3.7']

### ISSUE THREAD

**Issue by smoki3** [py-smart-gardena/hass-gardena-smart-system#192]
I actually have the problem I get the error:

```
Logger: homeassistant.setup
Source: setup.py:221
First occurred: 14:38:29 (1 occurrences)
Last logged: 14:38:29

Setup failed for custom integration 'gardena_smart_system': Requirements for gardena_smart_system not found: ['py-smart-gardena==1.3.7'].
```

I updated to 2024.b6 since this the integration is not working anymore

**Comment by dennisduysak** [comment]
I have exactly the same issue

**Comment by martinw72** [comment]
I expected issues as I saw they updated to a new python version.  Unfortunately it happened to some of my important integrations! Fu…

**Comment by martinw72** [comment]
logger:%20homeassistant.util.package%0ASource%3A%20util/package.py:122%0AFirst%20occurred:%2021:27:06%20(3%20occurrences)%0ALast%20logged:%2021:27:10%0A%0AUnable%20to%20install%20package%20py-smart-gardena==1.3.7:%20error:%20subprocess-exited-with-error%20%C3%97%20python%20setup.py%20egg_info%20did%20not%20run%20successfully.%20%E2%94%82%20exit%20code:%201%20%E2%95%B0%E2%94%80%3E%20%5B18%20lines%20of%20output%5D%20/tmp/pip-install-yd087bug/py-smart-gardena_55adaaa0ee41461f8360922073c98148/versioneer.py:421:%20SyntaxWarning:%20invalid%20escape%20sequence%20'%5Cs'%20LONG_VERSION_PY%5B'git'%5D%20=%20'''%20Traceback%20(most%20recent%20call%20last):%20File%20%22%3Cstring%3E%22,%20line%202,%20in%20%3Cmodule%3E%20File%20%22%3Cpip-setuptools-caller%3E%22,%20line%2034,%20in%20%3Cmodule%3E%20File%20%22/tmp/pip-install-yd087bug/py-smart-gardena_55adaaa0ee41461f8360922073c98148/setup.py%22,%20line%2016,%20in%20%3Cmodule%3E%20version=versioneer.get_version(),%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-yd087bug/py-smart-gardena_55adaaa0ee41461f8360922073c98148/versioneer.py%22,%20line%201480,%20in%20get_version%20return%20get_versions()%5B%22version%22%5D%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-yd087bug/py-smart-gardena_55adaaa0ee41461f8360922073c98148/versioneer.py%22,%20line%201412,%20in%20get_versions%20cfg%20=%20get_config_from_root(root)%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-yd087bug/py-smart-gardena_55adaaa0ee41461f8360922073c98148/versioneer.py%22,%20line%20342,%20in%20get_config_from_root%20parser%20=%20configparser.SafeConfigParser()%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20AttributeError:%20module%20'configparser'%20has%20no%20attribute%20'SafeConfigParser'.%20Did%20you%20mean:%20'RawConfigParser'?%20%5Bend%20of%20output%5D%20note:%20This%20error%20originates%20from%20a%20subprocess,%20and%20is%20likely%20not%20a%20problem%20with%20pip.%20error:%20metadata-generation-failed%20%C3%97%20Encountered%20error%20while%20generating%20package%20metadata.%20%E2%95%B0%E2%94%80%3E%20See%20above%20for%20output.%20note:%20This%20is%20an%20issue%20with%20the%20package%20mentioned%20above,%20not%20pip.%20hint:%20See%20above%20for%20details.%0AUnable%20to%20install%20package%20py-smart-gardena==1.3.7:%20error:%20subprocess-exited-with-error%20%C3%97%20python%20setup.py%20egg_info%20did%20not%20run%20successfully.%20%E2%94%82%20exit%20code:%201%20%E2%95%B0%E2%94%80%3E%20%5B18%20lines%20of%20output%5D%20/tmp/pip-install-mhdmpx1o/py-smart-gardena_10a18df0abe54939b8cb4ee98576450a/versioneer.py:421:%20SyntaxWarning:%20invalid%20escape%20sequence%20'%5Cs'%20LONG_VERSION_PY%5B'git'%5D%20=%20'''%20Traceback%20(most%20recent%20call%20last):%20File%20%22%3Cstring%3E%22,%20line%202,%20in%20%3Cmodule%3E%20File%20%22%3Cpip-setuptools-caller%3E%22,%20line%2034,%20in%20%3Cmodule%3E%20File%20%22/tmp/pip-install-mhdmpx1o/py-smart-gardena_10a18df0abe54939b8cb4ee98576450a/setup.py%22,%20line%2016,%20in%20%3Cmodule%3E%20version=versioneer.get_version(),%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-mhdmpx1o/py-smart-gardena_10a18df0abe54939b8cb4ee98576450a/versioneer.py%22,%20line%201480,%20in%20get_version%20return%20get_versions()%5B%22version%22%5D%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-mhdmpx1o/py-smart-gardena_10a18df0abe54939b8cb4ee98576450a/versioneer.py%22,%20line%201412,%20in%20get_versions%20cfg%20=%20get_config_from_root(root)%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-mhdmpx1o/py-smart-gardena_10a18df0abe54939b8cb4ee98576450a/versioneer.py%22,%20line%20342,%20in%20get_config_from_root%20parser%20=%20configparser.SafeConfigParser()%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20AttributeError:%20module%20'configparser'%20has%20no%20attribute%20'SafeConfigParser'.%20Did%20you%20mean:%20'RawConfigParser'?%20%5Bend%20of%20output%5D%20note:%20This%20error%20originates%20from%20a%20subprocess,%20and%20is%20likely%20not%20a%20problem%20with%20pip.%20error:%20metadata-generation-failed%20%C3%97%20Encountered%20error%20while%20generating%20package%20metadata.%20%E2%95%B0%E2%94%80%3E%20See%20above%20for%20output.%20note:%20This%20is%20an%20issue%20with%20the%20package%20mentioned%20above,%20not%20pip.%20hint:%20See%20above%20for%20details.%0AUnable%20to%20install%20package%20py-smart-gardena==1.3.7:%20error:%20subprocess-exited-with-error%20%C3%97%20python%20setup.py%20egg_info%20did%20not%20run%20successfully.%20%E2%94%82%20exit%20code:%201%20%E2%95%B0%E2%94%80%3E%20%5B18%20lines%20of%20output%5D%20/tmp/pip-install-gvx26zdl/py-smart-gardena_2c91d4f95c6d44db8108f85623047c9e/versioneer.py:421:%20SyntaxWarning:%20invalid%20escape%20sequence%20'%5Cs'%20LONG_VERSION_PY%5B'git'%5D%20=%20'''%20Traceback%20(most%20recent%20call%20last):%20File%20%22%3Cstring%3E%22,%20line%202,%20in%20%3Cmodule%3E%20File%20%22%3Cpip-setuptools-caller%3E%22,%20line%2034,%20in%20%3Cmodule%3E%20File%20%22/tmp/pip-install-gvx26zdl/py-smart-gardena_2c91d4f95c6d44db8108f85623047c9e/setup.py%22,%20line%2016,%20in%20%3Cmodule%3E%20version=versioneer.get_version(),%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-gvx26zdl/py-smart-gardena_2c91d4f95c6d44db8108f85623047c9e/versioneer.py%22,%20line%201480,%20in%20get_version%20return%20get_versions()%5B%22version%22%5D%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-gvx26zdl/py-smart-gardena_2c91d4f95c6d44db8108f85623047c9e/versioneer.py%22,%20line%201412,%20in%20get_versions%20cfg%20=%20get_config_from_root(root)%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20File%20%22/tmp/pip-install-gvx26zdl/py-smart-gardena_2c91d4f95c6d44db8108f85623047c9e/versioneer.py%22,%20line%20342,%20in%20get_config_from_root%20parser%20=%20configparser.SafeConfigParser()%20%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%5E%20AttributeError:%20module%20'configparser'%20has%20no%20attribute%20'SafeConfigParser'.%20Did%20you%20mean:%20'RawConfigParser'?%20%5Bend%20of%20output%5D%20note:%20This%20error%20originates%20from%20a%20subprocess,%20and%20is%20likely%20not%20a%20problem%20with%20pip.%20error:%20metadata-generation-failed%20%C3%97%20Encountered%20error%20while%20generating%20package%20metadata.%20%E2%95%B0%E2%94%80%3E%20See%20above%20for%20output.%20note:%20This%20is%20an%20issue%20with%20the%20package%20mentioned%20above,%20not%20pip.%20hint:%20See%20above%20for%20details.%0A

**Comment by grm** [comment]
Hello,

I will release a new version tomorrow that should solve the issue.

Jeremie

**Comment by grm** [comment]
I had issues yesterday with travisci which does not work as expected anymore.
I migrated to github actions for building the underlying library py-smart-gardena.

I just release a 1.1.0-rc3 version (in beta) that should solve the issue.

**Comment by Jpsy** [comment]
Works perfectly!
Many thanks for the quick action Jérémie.

**Comment by heikobau** [comment]
Many thanks also from me!

**Comment by martinw72** [comment]
Yes, everything fixed, perfect. Thanks a lot!

**Comment by HolgerM1** [comment]
Hi!
same issue here. I already downloaded the release 1.1.0-rc3. Could anybody give me a hint how to update thi in homeassistant?

thnaks!

**Comment by megaherb** [comment]
Thank you very much for the fast fix.

@HolgerM1 You'll find instructions here:

https://hacs.xyz/docs/faq/beta/

**Comment by grm** [comment]
Fixed in 1.1.0-rc3
